module-services/service-db/ServiceDBCommon.cpp

```diff
@@ -99,8 +99,8 @@ sys::ReturnCodes ServiceDBCommon::SwitchPowerModeHandler(const sys::ServicePower
     return sys::ReturnCodes::Success;
 }
 
-void ServiceDBCommon::sendUpdateNotification(db::Interface::Name interface, db::Query::Type type)
+void ServiceDBCommon::sendUpdateNotification(db::Interface::Name interface, db::Query::Type type, uint32_t recordId)
```

Comment:
Tbh I do not like the solution here. What if there will be multiple records changed?
what if records will be removed? what if these will be added? 
Is there a way to not change the general notifications logic in ServiceDB (used in multiple apps and services in the bell and pure) and add the required logic in ServiceDesktop?

Reply:
Will be changed in the next PR, as discussed.
For now, it stays as is.